### PR TITLE
feat(maintner): adds GetIssue and ForIssue functions

### PR DESCRIPTION
**Pull request by orthros**

### CODE REVIEW

File: maintner/github.go [hunk 1]
```diff
@@ -143,6 +143,31 @@ func (gr *GitHubRepo) ForeachMilestone(fn func(*GitHubMilestone) error) error {
 	return nil
 }
 
+// ErrIssueNotFound represents the case where an issue is not found.
+var ErrIssueNotFound = fmt.Errorf("issue not found.")
+
+// GetIssue returns a GitHubIssue with the given ID.
+//
+// It returns nil if no issue with the given ID exists
```
Comment (jmdobry) [inline maintner/github.go:151]: What if the issue is in the map but has been tombstoned (marked as not_exists)?

**Review by jmdobry — CHANGES_REQUESTED** [review]
Tests?

File: maintner/github.go (hunk 1 above)
Comment (orthros) [reply maintner/github.go:151]: Our downstream code handles that by also checking the .NotExist field. Could be added here as well

Comment (jmdobry) [reply maintner/github.go:151]: Okay, if `Foreachissue` returns NotExist issues then this should too.

**Review by jmdobry — APPROVED** [review]

**Pull request merged**